dashboard/app/main.go

```diff
@@ -1497,12 +1497,15 @@ func getUIJobs(keys []*db.Key, jobs []*Job) []*uiJob {
 
 func loadTestPatchJobs(c context.Context, bug *Bug) ([]*uiJob, error) {
 	bugKey := bug.key(c)
+
+	const maxJobs = 10
 	var jobs []*Job
 	keys, err := db.NewQuery("Job").
 		Ancestor(bugKey).
 		Filter("Type=", JobTestPatch).
 		Filter("Finished>=", time.Time{}).
 		Order("-Finished").
+		Limit(maxJobs).
```

Comment:
You want to limit the number of automatically created jobs, right?
Perhaps it's better to do it explicitly? I.e. in the loop below discard automatic jobs if we already have few.
Otherwise we can drop user-created jobs with interesting patches.

Reply:
Hmm, yes, it's indeed better. Updated the PR.